tests/unit/test_nvopen_202_recovery.py

```
@@ -0,0 +1,114 @@
+            (0, 10, 0, "20260209"),  # Retry: success
+        ]
+
+        with patch("src.nvlink.wrapper.NVLinkWrapper.__init__", return_value=None):
+            from src.nvlink.wrapper import NVLinkWrapper
+            wrapper = NVLinkWrapper.__new__(NVLinkWrapper)
+            wrapper._nvlink = mock_nvlink_com
+            wrapper._is_open = False
+            wrapper._com_initialized = True
+            wrapper.sid = "TEST"
+
+            result = wrapper.nv_open("RACE", "20260101000000", option=1)
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)